.vscode/settings.json

```diff
@@ -12,4 +12,7 @@
     "**/templates/**/*": "django-txt",
     "**/requirements{/**,*}.{txt,in}": "pip-requirements"
   },
+  "python.analysis.extraPaths": [
+    "./schemas/chronopost"
+  ],
```

Comment:
Instead of this. Please add the package to the dev dependency

`requirements.sdk.dev.txt`
```txt
# Schemas update
-e ./schemas/chronopost
```
same for `requirements.server.dev.txt`
```txt
# Schemas update
-e ./schemas/chronopost
```